main.tf

```diff
@@ -25,23 +25,24 @@ resource "aws_security_group" "this" {
     Env       = var.env
     Name      = local.name
   }
-  
+
   lifecycle {
     create_before_destroy = true
   }
 }
 
 # EC2
 resource "aws_instance" "this" {
-  ami                    = join("", data.aws_ami.ubuntu_20_04.*.id)
+  ami                    =  "ami-022d4249382309a48"
```

Comment:
Let's provide more information on what kind of AMI is it, and why we are locking it:ie
```
# Ubuntu 20.04.xx. If upgrade is needed, you would need to re-connect to openvpn.
```
(specify exact version of ubuntu)